pkg/detectors/databrickstoken/databrickstoken.go

```diff
@@ -21,52 +21,54 @@ var (
 	client = common.SaneHttpClient()
 
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
-	domain = regexp.MustCompile(`\b(https:\/\/[a-z0-9-]+\.cloud\.databricks\.com)\b`)
-	keyPat = regexp.MustCompile(`\b(dapi[a-z0-9]{32})\b`)
+	keyPat    = regexp.MustCompile(`\b(dapi[0-9Aa-f]{32})\b`)
```

Comment:
Are you sure this pattern is correct? `[0-9Aa-f]` is an unusual character set. I would expect either `[0-9A-Fa-f]` or `[0-9a-f]`.

Reply:
it should be `[0-9a-f]` that was a mistake, thank you for spotting that. I will fix it right away